pkg/kotsadmsnapshot/backup.go

```diff
@@ -379,6 +390,11 @@ func CreateInstanceBackup(ctx context.Context, cluster *downstreamtypes.Downstre
 		}
 	}
 
+	// excludeShutdownPodsFromBackup will add a hook to the backup to exclude pods that are in the process of shutting down
```

Comment:
this comment isn't necessary since the function definition already has the same comment